src/core/Delta.ts

```diff
@@ -31,32 +31,103 @@ export enum DeltaQueue {
   kUnknown
 }
 
+/**
+ * Output JSON format for delta.json and deltaLog.json based on feedback
+ * from the AWG on 8/22/2023 to keep the output simple
+ * 
+ * So internally, we are storing the full CveCorePlus, but externally,
+ * and specifically when writing out to JSON, we are using this shortened format
+ 
+ * see https://github.com/CVEProject/cvelistV5/issues/23 for some additional discussions
+ * before and after the AWG meeting on 8/22
+ */
+export class DeltaOutpuItem {
+
+  static _cveOrgPrefix = `https://www.cve.org/CVERecord?id=`;
+  static _githubRawJsonPrefix = `https://raw.githubusercontent.com/CVEProject/cvelistV5/main/cves/`
+
+  cveId: string;          // string version of the CVE ID
+  cveOrgLink?: string;    // url to cve.org record
```

Comment:
Was there a specific requirement to add this link into the data? The GitHub link is useful since it can be used to download the specific record file.

Reply:
The Github prefix was there in the beginning of this task as a way to leverage Github's role in downloading CVEs.  It is a automatic benefit from using the Github repository, as well as the hourly release (zip) artifacts.  It also guarantees that the data would be identical to a `git pull` command.

The other one on the line right above it
```
static _cveOrgPrefix = `https://www.cve.org/CVERecord?id=`
```
came about from the AWG discussion 2 weeks ago to put a `cve.org` link in addition to the Github link.